### ISSUE TITLE
Idea: Different styles for album covers

### ISSUE THREAD

**Issue by ghost** [kmikiy/SpotMenu#40]
I really love how album cover art is displayed currently. 
But I wonder, if you could add some styles for customisation. 
For example rounded edges for the artwork or round artwork (vinyl style). 
Think this could look great :)

**Comment by fabi94music** [comment]
A made a poorly drafted sketch of what I meant with rounded corners :)
![draft rounded corners and shadow cover art](https://user-images.githubusercontent.com/29753374/28546057-990cc85a-70c9-11e7-8148-950f7378a728.jpg)

**Comment by ken89llo** [comment]
This looks great! exactly what I was looking for.

**Comment by ken89llo** [comment]
Think this would look great with a animation like in the Spotify for iOS app when switching to the next track.

**Comment by fabi94music** [comment]
This is how MiniPlay behaves. Quite nice. But animation might be difficult. 


![ezgif com-crop](https://user-images.githubusercontent.com/29753374/28550054-b6e56bba-70de-11e7-8d82-55b546d2f95f.gif)

**Comment by ken89llo** [comment]
yeah I like the animation a lot, makes it even more aesthetically pleasing.

**Comment by fabi94music** [comment]
Maybe this could be helpful 🙈 
https://github.com/PierrePerrin/PPMusicImageShadow

**Comment by fabi94music** [comment]
I like that fork :) https://github.com/andreamazz/SpotMenu

**Comment by kmikiy** [comment]
[SpotMenu.app.zip](https://github.com/kmikiy/SpotMenu/files/1565696/SpotMenu.app.zip)
i've added a corner radius to notification widget
<img width="361" alt="screen shot 2017-12-17 at pm 2 38 45" src="https://user-images.githubusercontent.com/14206503/34080007-0180a772-e338-11e7-8869-c0be922c9751.png">
